feature-libs/product-configurator/common/components/configurator-cart-entry-bundle-info/configurator-cart-entry-bundle-info.model.ts

```diff
@@ -0,0 +1,8 @@
+/**
+ * An interface representing the CPQ Line Item from order entry.
+ */
+export interface LineItem {
+  name?: string;
```

Comment:
Name should not be optional

Reply:
Why do you think so?

Reply:
What is the advantage of it?

Reply:
Sascha already answered this question. There is a case where a name is not present and we still have a valid line item.